src/NextGen/DonationForm/Controllers/DonateController.php

```diff
@@ -91,17 +93,22 @@ private function getOrCreateDonor(
      *
      * @unreleased
      *
-     * @param $donationId
-     *
      * @return void
      */
-    private function setSession($donationId)
+    private function setSession(Donation $donation, Donor $donor)
```

Comment:
Let's update this docblock to include our future intentions withs sessions, to give our future selves peace about removing this.

Reply:
👍 280085d5